draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -330,17 +330,16 @@ congestion control.
 | Volatile paths               | yes                   | yes                |
 | Session handover             | yes                   | yes                |
 | Path aggregation             | yes                   | yes                |
-| Robust session establishment | no                    | yes                |
 | Data reordering              | yes                   | optional / modular |
 | Expandability                | limited by TCP header | flexible           |
 {: #table_mptcp_mpdccp_comp title='MPTCP and MP-DCCP protocol comparison'}
 
 Therefore, the sender logic is not much different between MP-DCCP and
 MPTCP, even if the multipath session initiation differs.  MP-DCCP
-inherits a robust session establishment feature, which guarantees
-communication establishment if at least one functional path is
-available.  MPTCP relies on an initial path, which has to work;
-otherwise no communication can be established.
+inherits a faster establishment of subsequent subflows already during the setup of the
+initial subflow.  MPTCP has more constraints on the initial path setup, which
+has to pass a 4-way handshake first; This leaves MPTCP with higher setup times of
+multiple flows and less robust against delays and packet loss on the initially selected path.
```

Comment:
I think the added new text is quite hard to parse. Are the details important? I think we said that we would anyway remove this part later so maybe we can just say that the multipath session initiation differs and leave it at that? At least for now to get the text merged. Then we can expand again later if needed.

I think the new text may just raise questions. What does "inherits a faster establishment" mean? Just before the table we say that "Both transport protocols share the 3-way initiation of a communication" so this does not suggest that DCCP is faster? 
Also, the description of MP-DCCP suggests that if the MP-DCCP setup messages fail over the selected path it will fall back to DCCP, so I do not understand why this is more robust than for MPTCP? MPTCP will also fall back to TCP if the option is not supported.